CourseGrab/Home/HomeTableViewCell.swift

```diff
@@ -8,7 +8,152 @@
+            sectionLabel.text = section.section
+            statusBadge.image = section.status.icon
+            titleLabel.text = section.title
+
+            if section.status == .open {
+                removeButton.snp.makeConstraints { make in
+                    make.top.equalTo(sectionLabel.snp.bottom).offset(12)
+                    make.leading.equalTo(16)
+                    make.height.equalTo(24)
+                    make.trailing.equalTo(container.snp.centerX).inset(9)
+                    make.bottom.equalTo(container).inset(16)
+                }
+            } else {
+                removeButton.snp.remakeConstraints { make in
+                    make.top.equalTo(sectionLabel.snp.bottom).offset(12)
+                    make.leading.equalTo(16)
+                    make.height.equalTo(24)
+                    make.trailing.equalTo(container).inset(16)
+                    make.bottom.equalTo(container).inset(16)
+                }
+            }
+        }
+    }
+    
+    private let container = UIView()
+    private let courseLabel = UILabel()
+    private let enrollButton = UIButton(type: .roundedRect)
+    private let removeButton = UIButton(type: .roundedRect)
+    private let sectionLabel = UILabel()
+    private let statusBadge = UIImageView()
+    private let titleLabel = UILabel()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        // setup appearance
+        
+        selectionStyle = .none
+        
+        container.translatesAutoresizingMaskIntoConstraints = false
+        container.clipsToBounds = false
+        container.layer.cornerRadius = 5
+        container.layer.shadowColor = UIColor.black.cgColor
+        container.layer.shadowOpacity = 0.3
+        container.layer.shadowRadius = 2
+        container.layer.shadowOffset = .zero
+        container.backgroundColor = .white
+        contentView.addSubview(container)
+        
+        courseLabel.translatesAutoresizingMaskIntoConstraints = false
+        courseLabel.font = .boldSystemFont(ofSize: 14)
+        courseLabel.textColor = .courseGrabGray
+        container.addSubview(courseLabel)
+        
+        enrollButton.translatesAutoresizingMaskIntoConstraints = false
+        enrollButton.layer.cornerRadius = 2
+        enrollButton.setTitleColor(.white, for: .normal)
+        enrollButton.setTitle("ENROLL", for: .normal)
+        enrollButton.titleLabel?.font = .systemFont(ofSize: 12)
+        enrollButton.backgroundColor = .black
+        container.addSubview(enrollButton)
+        
+        removeButton.translatesAutoresizingMaskIntoConstraints = false
+        removeButton.layer.cornerRadius = 2
+        removeButton.setTitleColor(.courseGrabRuby, for: .normal)
+        removeButton.setTitle("REMOVE", for: .normal)
+        removeButton.titleLabel?.font = .systemFont(ofSize: 12)
```

Comment:
I believe we have a font for this in Fonts extensions so we should use that for consistency (which is what I used).